pkg/log/log_test.go

```diff
@@ -5,12 +5,22 @@ package log
 import (
 	"testing"
 
+	crdv1 "github.com/kubernetes-incubator/external-storage/snapshot/pkg/apis/crd/v1"
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	appv1 "k8s.io/api/apps/v1"
+	appv1beta1 "k8s.io/api/apps/v1beta1"
+	appv1beta2 "k8s.io/api/apps/v1beta2"
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestLog(t *testing.T) {
 	t.Run("podLogTest", podLogTest)
+	t.Run("deploymentLogTest", deploymentLogTest)
```

Comment:
Oh, nice use of golang 1.7 subtests: https://blog.golang.org/subtests

I've been working a lot with px-test, which was started with an older version of go, so doesn't use these techniques for running a suite of tests.